arbigent-cli/src/main/kotlin/VariableParser.kt

```
@@ -0,0 +1,175 @@
+            }
+        }
+        
+        // Validate variable names
+        jsonMap.forEach { (name, _) ->
+            if (!isValidVariableName(name)) {
+                throw CliktError("Invalid variable name: '$name'. Variable names must start with a letter or underscore and contain only letters, numbers, and underscores.")
+            }
+        }
+        
+        return jsonMap
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)